docs/extensibility/form-section.md

```diff
@@ -45,6 +46,16 @@ If you target elements of an array rather than the array itself, you can use the
       name: Item name
     - path: service.url
     - path: service.port
+    - path: service.number
```

Comment:
you added a mutually exclusive `visibility` and yet you didn't include `overwrite` here. Which should be used.
we could use an example with single `visibility` and no `overwrite` in addition to it as well